ichnaea/webapp/app.py

```diff
@@ -2,10 +2,16 @@
 Holds global web application state and the WSGI handler.
 """
 
-from wsgiref.simple_server import make_server
+import logging
+
+from waitress import serve
```

Comment:
I swapped wsgiref out for waitress which pyramid depends on already. It's more featured and probably better to use.